src/main/java/qna/domain/AnswersRelatedQuestion.java

```diff
@@ -0,0 +1,34 @@
+package qna.domain;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+@Embeddable
+public class AnswersRelatedQuestion {
```

Comment:
다른 도메인과의 협력을 생각해보면 클래스 이름이 Answers가 더욱 적합해보이기도 하는데 어떻게 생각하시나요 😄 

```suggestion
public class Answers {
```